src/App.js

```diff
@@ -41,6 +42,11 @@ function App() {
           </Row>
         </Container>
       </main>
+      <Container className="footer">
```

Comment:
This should live inside of a `<footer>` tag. Ideally remove the footer class. And there needs to be a `<Col xl={8}` inside the row and then call the Footer